lighthouse-core/audits/audit.js

```diff
@@ -162,18 +163,26 @@ class Audit {
 
     // If the audit was determined to not apply to the page, we'll reset it as informative only
     if (result.notApplicable) {
-      score = 1;
       scoreDisplayMode = Audit.SCORING_MODES.NOT_APPLICABLE;
       result.rawValue = true;
     }
 
+    if (result.error) {
```

Comment:
is it possible to rework this code to be a flow through a set of mutually exclusive states so it's easy to see the possibilities?

e.g. score is null, if `error`, it's always scoring mode error, else if it's been marked `notApplicable`, it's mode `notApplicable`, then `manual`, and then only if it's binary or numeric is a score calculated?

`description` seems like it might not fit the mold, but then maybe it's worth calling separately if it sits across the display modes differently than `score` does

Reply:
this gets reworked immediately as part of #5132, fair to take up there?

Reply:
> fair to take up there?

yeah, SG